delivery_carrier_label_gls/README.md

```diff
@@ -0,0 +1,33 @@
+
+GLS Group - Transportation services
+=====================================
+
+Implémentation du service Unibox service implementation:
+- send delivery order informations and parcel infothe GLS unibox server
+- webservice routing info in back
+- label GLS Unibox generate
+
+
+TODO:
+- translation
+- Shop Delivery Service
+- delivery Express Parcel Guaranteed (for companies only)
+- migrate the module in 8 version
+
+Pull Request are welcomed to gather improvement on this module
+
+### Account GLS settings by company
+![Account GLS settings by company Odoo ERP](/delivery_carrier_label_gls/static/description/gls1.png)
+
+
+### Odoo Delivery Order avec GLS carrier
+![Odoo Delivery Order avec GLS carrier with Odoo ERP](/delivery_carrier_label_gls/static/description/gls2.png)
+
+
+### GLS transport label towards France generated with Odoo ERP
+![GLS transport label towards France generated with Odoo ERP](/delivery_carrier_label_gls/static/description/gls3.png)
+
+
+### GLS transport label towards foreign countries
+![GLS transport label towards foreign countries](/delivery_carrier_label_gls/static/description/gls4.png )
+
```

Comment:
For consistence with other OCA modules could you use reStructuredText instead of markdown?

Here is the template:

https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst